### PR TITLE
Add aux value element

### PR DESCRIPTION
**Pull request by haslinghuis**
Adds element from https://github.com/betaflight/betaflight/pull/10789

### CODE REVIEW

File: locales/en/messages.json
Comment (ctzsnooze) [inline locales/en/messages.json:4985]: Maybe "Displays a receiver AUX channel see PR 10789"

Comment (haslinghuis) [reply locales/en/messages.json]: Done.

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Comment by haslinghuis** [comment]
Rebased

**Review by blckmn — APPROVED** [review]

**Pull request merged**
